packages/components/src/internal/actions.ts

```diff
@@ -1813,126 +1844,59 @@ function applySelection(
     };
 }
 
-export function initLookup(column: QueryColumn, maxRows: number, values?: List<string>, keys?: List<any>) {
-    if (shouldInitLookup(column, values)) {
-        const store = new LookupStore({
-            key: LookupStore.key(column),
-            isLoaded: false,
-            isLoading: true,
-        });
-        updateLookupStore(store, {}, false);
+export const findLookupValues = async (
```

Comment:
I don't believe this needs to be exported. I think the only usages are in this file.